RebeccaJasper/Lab-Project-19G06
# Separate App Routing

Landing page and routing changed so that users navigate through Identi.Kreate and Identi.Match independently.

## Features
- Landing page at base URL has options to visit Identi.Kreate and Identi.Match apps.
- If  Identi.Kreate is selected, users are taken straight to the /statement page (followed by the /identi-kreate page and the /complete page).
- The /statement page has been modified to accommodate more applications of the app and more detailed questioning.
- The photo_id has been removed from the submission object (since photos are only part of the data collection app).
- No pages from the data collection version of Identi.Kreate have been removed - routing merely bypasses these pages. The original landing page for data collection (home.html) can be found at the /identi-fi URL.
- If Identi.Match is selected, the user is taken to the /submissions URL.

## Testing PR
- Please check that the routing works as described.
- Check the content of the /statement page and ensure that the respective text areas display based on the radio buttons or selections made by the user.
- Try submit a face by commenting out the AJAX request (so that no database call is made with fake data) and check that no errors relating to a photo_id not being present occur.
